tests/unit/utils/containers.go

```diff
@@ -0,0 +1,48 @@
+package utils
+
+import (
+	cassdcV1beta1 "github.com/datastax/cass-operator/operator/pkg/apis/cassandra/v1beta1"
+	. "github.com/onsi/gomega"
+	coreV1 "k8s.io/api/core/v1"
+)
+
+func GetInitContainer(cassdc *cassdcV1beta1.CassandraDatacenter, name string) *coreV1.Container {
+	return GetContainerByName(cassdc.Spec.PodTemplateSpec.Spec.InitContainers, name)
+
+}
+
+func GetContainer(cassdc *cassdcV1beta1.CassandraDatacenter, name string) *coreV1.Container {
+	return GetContainerByName(cassdc.Spec.PodTemplateSpec.Spec.Containers, name)
+
+}
+
+func AssertInitContainerNamesMatch(cassdc *cassdcV1beta1.CassandraDatacenter, names ...string) {
```

Comment:
I was just thinking about my previous two comment. Maybe it makes more sense to put CassandraDatacenter-specific function into their own package, e.g., `cassdc`. Then we can just write `cassdc.GetContainer`. vs `GetCassdcContainer`? 

wdty?

Note that I am using `cassdc` because it is the abbreviation specified in the CassandraDatacenter CRD. It allows to me to do `kubectl get cassdc` for example.

Reply:
I like the idea of having the functions in a separate package. It improves the focus of the package, shortens function names without losing clarity, and in places where you only need one, you can dot-alias the imports and not qualify them at all.

I made the change locally to move the cassdc functions to a separate package, but then found that `cassdc` is a very common (and reasonable) variable name, and using `cassdc` as the import alias creates a conflict. The conflict doesn't cause a compile failure; the variables just win. I conclude that `cassdc` is not a good choice for the alias. I changed it to use a dot alias in template_cassdc_test, and I think I like the result even more. :)